students/nick_miller/lesson03/mailroom.py

```diff
@@ -43,26 +168,13 @@ def report(db=donor_db):
           f"{key[2]:^10}",
           f"{separator:^3}",
           f"{key[3]:>15}")
-    print("-"*76)
-
-    for i in range(0, (len(donor_db))):
-        entry = (donor_db[i])
-        name = entry[0]
-        # print(name)
-        dons = entry[1]
-        totes = sum(dons)
-        # print(totes)
-        nums = len(dons)
-        # print(nums)
-        aves = totes/nums
-        # print(aves)
-        print(f"{name:<18}", f"{separator:^3}", f"{totes:>18.2f}", f"{separator:^3}", f"{nums:^10}", f"{separator:^3}",
-              f"{aves:>15.2f}")
-    print()
+    print("-" * 76)
+    for row in report_don:
+        print("{:<18s}   {:>15.2f}   {:12d}   {:>20.2f}".format(*row))
 
 
 def quit_prog():
-    print("quitting - see you next time")
+    print("Quitting - See you next time.")
     quit()
```

Comment:
This isn't actually quit the program - is it finished?

Reply:
Updated to import sys and properly quit. Will re-submit on next PR. Thanks!